agentMET4FOF/dashboard/Dashboard_Control.py

```diff
@@ -5,16 +5,19 @@
 
 
 #global variables access via 'dashboard_ctrl'
-class Dashboard_Control():
-    def __init__(self, ip_addr="127.0.0.1", port=3333, modules= []):
-        super(Dashboard_Control, self).__init__()
+class _Dashboard_Control():
```

Comment:
There is actually no such thing as a private class in a separate module according to PEP 8, as far as I know, although the underscore still clarifies, that we should not use this class for anything on its own. I think this should rather be refactored to be part of `Dashboard.py`.